frontend/src/app/modules/work-space/work-space-routing.module.ts

```diff
@@ -21,6 +22,7 @@ import { DashboardComponent } from './dashboard/dashboard.component';
           {
             path: 'dashboard',
             component: DashboardComponent,
+            canActivate: [AuthGuard],
```

Comment:
You don't have to use it here since it should already be applied to the parent role

Reply:
I removed this line